scipy/linalg/tests/test_lapack.py

```diff
@@ -1645,3 +1645,50 @@ def test_getc2_gesc2():
         else:
             assert_array_almost_equal(desired_cplx.astype(dtype),
                                       x/scale, decimal=4)
+
+
+def test_gttrf_gttrs():
+    np.random.seed(42)
+    n = 10
+    for index, dtype in enumerate(DTYPES):
+        atol = 10**-(np.finfo(dtype).precision-1)
```

Comment:
I'd suggest changing `rtol`, instead. For instance,
```
rtol=25*np.finfo(dtype).eps
```
appears 12 times in https://github.com/scipy/scipy/blob/master/scipy/linalg/tests/test_lapack.py.

I think it makes more sense because ultimately the reason we have to change a tolerance is that the _relative_ tolerance of single precision arithmetic is worse than double precision arithmetic.